frontend/src/app/page.tsx

```diff
@@ -1,7 +1,28 @@
+import LandingPageNav from "./components/LandingPage/LandingPageNav"
+import NewWorldText from "./components/LandingPage/HeadingTexts"
+import Paragraphs from "./components/LandingPage/Paragraphs"
+
 export default function Home() {
   return (
-    <div className="bg-red-500">
-      Test
+    <div
+     className="min-w-screen min-h-screen"
+     style={{
+      backgroundImage: `url('/handstouching_imrpoved.jpeg')`,
```

Comment:
is the original handstouching file used anywhere in the code? what is the purpose of having both the original and the improved in the repo?